plugin.video.longnow/changelog.txt

```diff
@@ -0,0 +1,22 @@
+v0.1.2 (2020-12-28)
```

Comment:
note: the changelog is no longer parsed by kodi. You might want to define the `<news>` element in addon xml addon metadata

Reply:
thanks, I added a news section as you suggested.